### ISSUE TITLE
Client disconnect on sending long text message

### ISSUE THREAD

**Issue by clehner** [umurmur/umurmur#172]
When I send a text message of some size, I get disconnected.
Data points: sending an image file of size 4741 (6799 chars source text) is okay, but a file of size 8454 (12142 chars source text) result in a disconnect.

**Comment by fatbob313** [comment]
Yeah that is 'by design'. The receive/send buffer for each client is 8 kb (8192 bytes) long. If a message larger than that is about to be received, the client is just disconnected, which seems a bit harsh I agree. The story behind that is the switch from Mumble 1.1.x and 1.2.x and the protocol changed. In short, old clients would talk the old protocol, which was interpreted as a really large sized message. That could possibly be handled better as well.
This small buffer size makes sense when running on memory constrained systems, which was the case 10 years ago when I chose that size. I guess that could be raised to at least 64 kb. I will think about it.

**Comment by MarcS1975** [comment]
Yes it would be helpful to increase buffer size or even allow for a config parameter. Mostt servers nowadays have 4 to 16GB ram and uploading a 5 MB document should not cause any problems.

**Comment by sethidden** [comment]
I think the premise of >>**u**<<Murmur is that it's meant to run on memory constrained systems like routers. Even for those released in 2022, they'd probably have 256, 512MB on the low-end. If your server has 4 or 16 GB you should be fine with using the regular `murmur` (also called mumble-server) that comes with the Mumble install. That isn't an issue there

**Comment by troxor** [comment]
Let's use this issue to test out Github discussions! It's an interesting consideration trying to balance memory constraints with the practicality of allowing long messages without forcing a disconnect.
